src/views/Home.vue

```diff
@@ -57,7 +57,8 @@
           </ion-item>
     
           <div class="scroller-content">
-            <div class="scroller-item" v-for="i = 1 in 10" :key="i">  
+            <div class="scroller-card" v-for="i = 1 in 10" :key="i"> 
```

Comment:
why do we need an extra container?

Reply:
When I went through the concept of scroll-snapping, I explored that scroll snapping is used by setting the `scroll-snap-type` property on a container element and the `scroll-snap-align` property on elements inside it. Here I have a container named `scroller-content` on which I set the `scroll-snap-type` then when I was setting the `scroll-snap-align: start` on child i.e. `scroller-item` here, It was moving itself at start of the container without scrolling. So I went through https://www.apple.com/store to see how they have used scroll snapping and then I followed it.

Reply:
If you leave scroll snap align to its default value you will not need this extra container.

Reply:
There is no default value of `scroll-snap-align` set on the `scroller-item`. If I am not giving any value to `scroll-snap-align`, scroll snapping is not working. So I tried giving it `start end` value (https://docs.w3cub.com/css/scroll-snap-align) and it is looking like this:

https://user-images.githubusercontent.com/82817616/155300610-1cd5db4c-2d1b-4668-8c99-c92039e68493.mp4